modules/azure/README.md

```diff
@@ -116,6 +116,31 @@ module "azure-managed-cloud" {
     module.azure-sn-cloud-manager
   ]
 }
+
+output "client_id" {
+  value       = module.azure-managed-cloud.sn_automation_client_id
+  description = "The client ID of the sn automation service principal for StreamNative Cloud automation"
+}
+
+output "support_client_id" {
+  value       = module.azure-managed-cloud.sn_support_client_id
+  description = "The client ID of the sn support service principal for StreamNative Cloud support access"
+}
+
+output "subscription_id" {
+  value       = module.azure-managed-cloud.subscription_id
+  description = "The subscription ID of the AKS cluster"
+}
+
+output "tenant_id" {
+  value       = module.azure-managed-cloud.tenant_id
+  description = "The tenant ID of the AKS cluster"
```

Comment:
could share the `tenant` concept of AKS?

Reply:
a tenant represents a dedicated and trusted instance of Azure Active Directory, identities (person, company, organization, service principle, etc...) need to be created in a tenant.

Reply:
customer should already know this concept if they are using Azure